test/system_test.py

```diff
@@ -403,6 +382,7 @@ def test_install_nwp_test_cpu_cce(self):
             'icon',
             build_on_login_node=True)
 
+    @pytest.mark.no_daint  # problem with gt4py and spack v21.1
```

Comment:
This deactivates the test on all machines. Please remove the test.

Reply:
done